ar.xjuan.Cambalache.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "ar.xjuan.Cambalache",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "separate-locales": false,
+    "command" : "cambalache",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
out of interest, what do you need that the portal can't give you via GtkFileChooserNative?

Reply:
Hi Nick, Opening a project and importing a UI file would work, but Cambalache supports multiple UI files in the same project which can then be exported to multiple .ui files and I did not had the time to investigate if its possible to do that with the file portal.